labs-ios-starter/Surveys/View Controllers/TopicViewController.swift

```diff
@@ -27,6 +27,35 @@ class TopicViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        fetchTopics()
+    }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == .getSegueID(.topicDetailSegue) {
+            guard let topicDetailViewController = segue.destination as? TopicDetailViewController else {
+                print("Invalid Segue for presenting Topic Detail")
+                return
+            }
+            guard let selectedIndex = topicsCollectionView.indexPathsForSelectedItems?.first else {
+                print("couldn't get selected index")
+                return
+            }
+            guard let topic = self.topics?[selectedIndex.item] else {
+                print("no topic")
+                return
+            }
+
+            //TESTING, REMOVE
+            let member = Member(id: "1", email: "1@1.com", firstName: "firstOne", lastName: "lastOne", avatarURL: URL(string: "http://www.url.com"))
```

Comment:
Testing comments

Reply:
we can comment it out for now if you want, but should be left as an example until we have a convenience method in place